docs/developer/deploy/upgrade.rst

```diff
@@ -0,0 +1,57 @@
+Upgrade a Platform with the latest dev build
+============================================
+
+Prerequisites
+-------------
+- A Platform already installed with all pod up & running
+- A new metalk8s.iso image with a higher version
+
+Procedure
+---------
+
+If upgrading from a lower patch, minor or major version just follow the
+`standard upgrade procedure <https://metal-k8s.readthedocs.io/en/latest/operation/upgrade.html>`_.
+
+If upgrading from the same patch, minor and major version:
+
+1. Upload the new metalk8s.iso on the bootstrap node
+
+2. Locate how metalk8s.iso is mounted
+
+  .. code-block:: console
+
+    [root@zenkotda-bootstrap centos]# cat /etc/fstab | grep metalk8s.iso
+    /root/metalk8s.iso		/srv/scality/metalk8s-2.6.0-dev	iso9660	nofail,ro	0 0
+
+3. Unmount the current metalk8s.iso
+
+  ``umount /srv/scality/metalk8s-2.6.0-dev``
+
+4. Copy the new metalk8s.iso in place of the old one
+
+  ``cp metalk8s.iso /root/metalk8s.iso``
+
+5. Mount the new metalk8s.iso
+
+  ``mount /root/metalk8s.iso /srv/scality/metalk8s-2.6.0-dev``
+
+6. Retrieve salt-master CONTAINER ID on bootstrap node
+
+  .. code-block:: console
+
+    [root@zenkotda-bootstrap centos]# crictl ps | egrep 'ID|salt-master'
+    CONTAINER ID        IMAGE               CREATED             STATE               NAME                       ATTEMPT             POD ID
+    b9a78b65322c0       c09887cdd144d       20 hours ago        Running             salt-master                3                   66f3c727f5b53
+
+7. Stop the salt-master container
+
+  ``crictl stop b9a78b65322c0``
```

Comment:
I think you can combine steps 6 and 7 with something like

6. From the Bootstrap node, stop the salt-master container

``salt_master=$(crictl ps -q --label io.kubernetes.container.name=salt-master --state Running) | crictl stop $salt_master``

Reply:
done